scripts/systemtap/kem/kemd.c

```diff
@@ -150,48 +153,56 @@ int kemd_rbs_init(struct kem_rb *rbs)
 static int kemd_init_module(void)
 {
 	dev_t devno;
-	int   err = 0;
```

Comment:
The original code is better.
Please read "doc/coding-style".
The variable names should be aligned.

Reply:
The same comment as above applies here

Reply:
```
67   * align identifier names, not asterisks or other type-declaration related
 68     decorations:
 69 
 70           struct foo {
 71                   const char        *f_name;
 72                   uint32_t           f_id;
 73                   const struct bar  *f_bar;
 74                   int              (*f_callback)(struct foo *f, int flag);
 75           };
 76 
 77       This rule applied to block-level variable declarations too.

```